http/handler_p2p.go

```diff
@@ -428,28 +428,28 @@ func (h *p2pHandler) bindRoutes(router *Router) {
 	syncDocuments.Responses.Set("400", errorResponse)
 	syncDocuments.Responses.Set("500", errorResponse)
 
-	syncCollectionsRequestSchema := openapi3.NewObjectSchema().
+	syncCollectionVersionsRequestSchema := openapi3.NewObjectSchema().
 		WithProperty("versionIDs", openapi3.NewArraySchema().WithItems(openapi3.NewStringSchema())).
 		WithProperty("timeout", openapi3.NewStringSchema())
 
-	syncCollectionsRequest := openapi3.NewRequestBody().
+	syncCollectionVersionsRequest := openapi3.NewRequestBody().
 		WithRequired(true).
-		WithContent(openapi3.NewContentWithJSONSchema(syncCollectionsRequestSchema))
+		WithContent(openapi3.NewContentWithJSONSchema(syncCollectionVersionsRequestSchema))
 
-	syncCollectionsResponse := openapi3.NewResponse().
-		WithDescription("Document sync completed successfully")
+	syncCollectionVersionsResponse := openapi3.NewResponse().
+		WithDescription("Collection fetch completed successfully")
```

Comment:
todo: This wording is inconsistent with the command name and the CLI documentation, please use `synchronize` instead of `fetch`.

Same on line 443.